src/ut_kvp.c

```diff
@@ -61,6 +61,7 @@ static size_t write_memory_callback(void *contents, size_t size, size_t nmemb, v
 static struct fy_node* process_include(const char *filename, int depth, struct fy_document *doc);
 static void merge_nodes(struct fy_node *mainNode, struct fy_node *includeNode);
 static void remove_include_keys(struct fy_node *node);
+static void parse_edid_bytes(const char *hex_str, unsigned char *bytes, size_t *byte_len);
```

Comment:
edid has nothing todo with anything.

that's just the name of the field... it's all data bytes.